xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentfield.vm

```diff
@@ -42,6 +44,11 @@
   #set ($value = '')
 #end
 
+## Forces the value of the comment to the requested content.
+#if ($content)
```

Comment:
Why not checking directly $request.content there?

Reply:
I usually map request values to local variables. But in this case since it is only used once I can replace it with `$request.content`

Reply:
Apparently you marked this one as resolve but didn't change it.